docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+                - set the default to a blank list: `power_validation_pts_dict.setdefault([[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]], [])`
+                - append the power validation point to the list at this given: `power_validation_pts_dict[[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]].append(power_validation_point.result)`
+
+            - create a dictionary of the expected capacities for use in the power validation check: `given_capacities = {}`
+            - the following lines or logic do the capacity validation check:
+            - create a list of non-matching capacity validation points: `non_matching_capacity_validation_points = []`
+            - create a list of missing capacity validation points: `missing_capacity_validation_points = []`
+            - loop through the expected_chwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ecwt_temps: `for ect in expected_ecwt_temps:`
+                    - look for the capacity validation point in capacity_validation_pts_dict: `if capacity_validation_pts_dict[[cwt, ect]]:`
+                        - calculate the expected capacity by multiplying the result of the formula by the rated_capacity: `expected_capacity = (cap_f_t_coefficients[0] + cap_f_t_coefficients[1] * cwt + cap_f_t_coefficients[2] * cwt^2 + cap_f_t_coefficients[3] * ect + cap_f_t_coefficients[4] * ect^2 + cap_f_t_coefficients[5] * cwt * ect) * rated_capacity`
+                        - get the given capacity: `given_capacity = capacity_validation_pts_dict[[cwt, ect]]`
+                        - add the given capacity to the given_capacities dictionary: `given_capacities[[cwt,ect]] = given_capacity`
+                        - Compare the expected capacity to the given capacity.  This should not be an exact match, but with a margin of error (see notes at the bottom for suggested margins of errors).  We don't need to do anything if the capacities match, but if they don't match we need to add the conditions to the non_matching_capacity_validation_points list: `if expected_capacity != given_capacity: non_matching_capacity_validation_points.append({"CHWT": cwt, "ECWT": ect})`
+                    - otherwise this value doesn't exist, append these conditions to the missing_capacity_validation_points list: `else: missing_capacity_validation_points.append({"CHWT": cwt, "ECWT": ect})`
+
+            - the following lines or logic do the power validation check:
+            - create a list of non-matching power validation points: `non_matching_power_validation_points = []`
+            - create a list of missing power validation points: `missing_power_validation_points = []`
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
```

Comment:
Based on above on line 34 I think this is supposed to be for chwt in expected_chwt_temps"

Reply:
everything has been updated to be ect and cwt

Reply:
Maybe I am not following these updates but the typical abbreviation for chilled water is CHW so I recommend that any variable that pertains to chilled water use CHW and any variable that represents condensing water be CW. Looks like they both use CW regardless of whether they are chilled or condenser water. This is a suggestion. Feel free to ignore.

Reply:
ah, I see what you're looking for now.  Using CHWT and ECT now